lib/quick_travel/resource_category.rb

```diff
@@ -4,5 +4,9 @@ module QuickTravel
   class ResourceCategory < Adapter
```

Comment:
This looks to be the only file you've not changed to use the `init_from_hash` file, is that intentional? If so I'd like to see a comment as to why

Reply:
Just re-read your commit message, I think this is because it actually has and end_point right? If that's the case then yeah, you don't need a comment and can ignore this one

Reply:
Yup that is why